Description: use dh-autoreconf
   * Non-maintainer upload.
   * Run dh-autoreconf to update config.* to build on ppc64el and
     update configure.ac and m4 to work with that. Closes: #759719
Author: Andreas Barth <aba@ayous.org>
Bug-Debian: https://bugs.debian.org/759719

Index: openexr/config/OpenEXRConfig.h.in.in
===================================================================
--- /dev/null
+++ openexr/config/OpenEXRConfig.h.in.in
@@ -0,0 +1,76 @@
+/* ./config/OpenEXRConfig.h.in.in.  Generated from configure.ac by autoheader.  */
+
+/* Define to 1 if you have <iomanip> header file */
+#undef HAVE_COMPLETE_IOMANIP
+
+/* darwin */
+#undef HAVE_DARWIN
+
+/* Define to 1 if you have the <inttypes.h> header file. */
+#undef HAVE_INTTYPES_H
+
+/* largestack */
+#undef HAVE_LARGE_STACK
+
+/* Define to 1 if you have the <limits.h> header file. */
+#undef HAVE_LIMITS_H
+
+/* procfs */
+#undef HAVE_LINUX_PROCFS
+
+/* Define to 1 if you have the <memory.h> header file. */
+#undef HAVE_MEMORY_H
+
+/* Define to 1 if you have the <stdint.h> header file. */
+#undef HAVE_STDINT_H
+
+/* Define to 1 if you have the <stdlib.h> header file. */
+#undef HAVE_STDLIB_H
+
+/* Define to 1 if you have the <strings.h> header file. */
+#undef HAVE_STRINGS_H
+
+/* Define to 1 if you have the <string.h> header file. */
+#undef HAVE_STRING_H
+
+/* Define to 1 if you have the <sys/stat.h> header file. */
+#undef HAVE_SYS_STAT_H
+
+/* Define to 1 if you have the <sys/types.h> header file. */
+#undef HAVE_SYS_TYPES_H
+
+/* Define to 1 if you have the <unistd.h> header file. */
+#undef HAVE_UNISTD_H
+
+/* Define to the address where bug reports for this package should be sent. */
+#undef PACKAGE_BUGREPORT
+
+/* Define to the full name of this package. */
+#undef PACKAGE_NAME
+
+/* Define to the full name and version of this package. */
+#undef PACKAGE_STRING
+
+/* Define to the one symbol short name of this package. */
+#undef PACKAGE_TARNAME
+
+/* Define to the home page for this package. */
+#undef PACKAGE_URL
+
+/* Define to the version of this package. */
+#undef PACKAGE_VERSION
+
+/* Define to 1 if you have the ANSI C header files. */
+#undef STDC_HEADERS
+
+/* Define to empty if `const' does not conform to ANSI C. */
+#undef const
+
+/* Define to `__inline__' or `__inline' if that's what the C compiler
+   calls it, or to nothing if 'inline' is not supported under any name.  */
+#ifndef __cplusplus
+#undef inline
+#endif
+
+/* Define to `unsigned int' if <sys/types.h> does not define. */
+#undef size_t
Index: openexr/configure.ac
===================================================================
--- openexr.orig/configure.ac
+++ openexr/configure.ac
@@ -404,3 +404,4 @@ AC_MSG_RESULT([
 ])
 
 
+AC_CONFIG_HEADERS([./config/OpenEXRConfig.h.in])
Index: openexr/m4/threads.m4
===================================================================
--- openexr.orig/m4/threads.m4
+++ openexr/m4/threads.m4
@@ -280,7 +280,7 @@ fi
 
 # Finally, execute ACTION-IF-FOUND/ACTION-IF-NOT-FOUND:
 if test x"$am_posix_sem_ok" = xyes; then
-        ifelse([$1],,AC_DEFINE(HAVE_POSIX_SEMAPHORES),[$1])
+        ifelse([$1],,AC_DEFINE(HAVE_POSIX_SEMAPHORES,[],[posix semaphores]),[$1])
         :
 else
         am_posix_sem_ok=no
